apps/ledger-live-desktop/src/renderer/screens/manager/AppsList/Item.tsx

```diff
@@ -85,9 +83,21 @@ const Item = ({
     state.apps,
   ]);
 
-  const ffKey = currencyId ? formatCurrencyIdToFeatureKey(currencyId) : "";
-  const feature = useFeature(ffKey as FeatureId);
-  const currencyFlagEnabled = !feature || feature.enabled;
+  const { getAllFlags } = useFeatureFlags();
+  const currencyFlags = getAllFlags();
```

Comment:
⚠️  i'm not familiar enough with firebase framework but are we sure this is not costly?
actually i'm really worried about the performance of this. 
- getAllFlags is implementing with a bunch of JSON.parse and it's fetching ALL flags where you instead want to only ask "is my currency concerned" (and you don't care about parsing the others).
- this is not secured by a useMemo, so this will run on each single re-render
- this will be running for each item in the AppsList then!🤔  

so we have AppsCount x FlagsCount number of JSON parse on each re-render.

Reply:
We can't add query parameters to only fetch currency flags on firebase. These are the only available methods: https://firebase.google.com/docs/reference/js/v8/firebase.remoteconfig.RemoteConfig#methods

My initial solution was to only parse the currency flags but an other PR comment recommended to do otherwise (https://github.com/LedgerHQ/ledger-live/pull/3320#discussion_r1184696787)

Indeed I should pass the flags to the items and not do the fetch from there I'll change that